acid/features/zuul_manager/manager.py

```diff
@@ -44,6 +52,24 @@ def _sanitize_args(self, pipeline, branch):
         sanitized_ref = shlex.quote(f'refs/head/{branch}')
         return sanitized_pipeline, sanitized_ref
 
+    def _gearman_conf_arg(self, conf_path=None):
+        if conf_path is None:
+            config = current_app.config
+            conf_path = config['zuul'].get('gearman_conf', None)
```

Comment:
So here, we should be using `self.gearman_conf`, which should be provided in manager construction.